umm-lib/src/cmr/umm/echo10/granule.clj

```diff
@@ -3,7 +3,7 @@
   (:require [clojure.data.xml :as x]
             [clojure.java.io :as io]
             [cmr.common.xml :as cx]
-            [cmr.umm.umm-granule :as g]
+            [cmr.umm.umm-granule :as g]    
```

Comment:
Looks like some whitespace snuck in here

Reply:
Oops. Thank you. I want to note that preventing this whitespace issue is the reason for the addition to the .gitignore file in this pull request. That fix, which involves adding a file for formatting rules, is outlined in the CMR Developer's Guide here: https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=228066700